tests/ttnn/unit_tests/operations/test_paged_update_cache.py

```diff
@@ -65,30 +86,53 @@ def run_test_update_cache_decode(
         eq_update, output_update = comp_pcc(x, tt_updated_slice)  # checks the updated parts
     logger.debug(output_cache)
     logger.debug(output_update)
+
+    if not eq_cache or not eq_update:
```

Comment:
we should remove this or change it to logger debug, definitely remove the breakpoint